src/commands/devchain.js

```diff
@@ -0,0 +1,132 @@
+const TaskList = require('listr')
+const ncp = require('ncp')
+const ganache = require('ganache-core')
+const Web3 = require('web3')
+const { promisify } = require('util')
+const os = require('os')
+const path = require('path')
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+const chalk = require('chalk')
+const fs = require('fs')
+
+const { BLOCK_GAS_LIMIT, MNEMONIC } = require('../ganache-vars')
+
+exports.command = 'devchain'
+exports.describe = 'Open a test chain for development and pass arguments to ganache'
+exports.builder = {
+  port: {
+    description: 'The port to run the local chain on',
+    default: 8545
+  },
+  reset: {
+    type: 'boolean',
+    default: false,
+    description: 'Reset devchain to snapshot'
+  },
+  accounts: {
+    default: 2,
+    description: 'Number of accounts to print'
+  },
+  verbose: {
+    default: false,
+    type: 'boolean',
+    description: 'Enable verbose devchain output'
+  }
+}
+
+exports.task = async function ({ port = 8545, verbose = false, reset = false, showAccounts = 2, reporter }) {
+  const removeDir = promisify(rimraf)
+  const mkDir = promisify(mkdirp)
+  const recursiveCopy = promisify(ncp)
+
+  const snapshotPath = path.join(os.homedir(), `.aragon/ganache-db-${port}`)
```

Comment:
We might have to figure out how to port this over from aragonCLI since we've since changed the snapshot directory to be postfixed by the version of the CLI.

One idea could be just to prefix the version instead to the version of aragen, since this is where the snapshot is actually coming from.

--------

Let's double check there's no other modifications to the part of aragonCLI that currently handles the devchain from November onwards.